azure-pipelines.yml

```
@@ -223,6 +223,8 @@ stages:
 
             - task: DotNetCoreCLI@2
               displayName: Dotnet generate SBOM
+              # Generating the Software Bill of Materials (SBOM)
+              # For more context about why this is needed, see: https://sonarsource.atlassian.net/browse/BUILD-1303 
```

I would do it only like this to keep it simple. The displayName already says SBOM and the link below has all the other necessary details.
```suggestion
              # https://sonarsource.atlassian.net/browse/BUILD-1303 
```